cmake/modules/SwiftConfigureSDK.cmake

```diff
@@ -429,14 +429,14 @@ macro(configure_sdk_unix name architectures)
 
         set(SWIFT_SDK_FREEBSD_ARCH_${arch}_TRIPLE "${arch}-unknown-freebsd${freebsd_system_version}")
       elseif("${prefix}" STREQUAL "OPENBSD")
-        if(NOT arch STREQUAL "amd64")
+        if(NOT arch STREQUAL "x86_64" AND NOT arch STREQUAL "aarch64")
```

Comment:
I see that we're switching from amd64 -> x86_64. I assume that's intentional?

Reply:
Yes. 

There are various parts of the codebase and build system, which basically assume architecture names with LLVM spellings, not platform spellings. If triples elsewhere are constructed with platform spellings, then missing module errors and the like end up manifesting. It ends up being simpler to just use the LLVM names rather than try and introduce special cases throughout the build system.

Reply:
That makes sense, thank you!